travis/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 echo "> Cloning FHIR Generated Patients..."
-git clone --single-branch --branch exm_506-recalculate https://github.com/projecttacoma/fhir-patient-generator.git
+git clone --single-branch --branch add-measure-version https://github.com/projecttacoma/fhir-patient-generator.git
```

Comment:
I guess do we need to change this back? Eventually master on FPG will be working. Or we can wait to merge this until the FPG PR gets merged